Cargo.toml

```diff
@@ -56,6 +56,7 @@ base64 = "0.22"
 bytes = "1.0"
 cargo_metadata = "0.18.1"
 clap = { version = "4.4.16", features = ["derive"] }
+cfg-if = "1.0.0"
```

Comment:
I used [`cfg-if`](https://docs.rs/cfg-if/latest/cfg_if/) to simplify some conditional compilation in `azure_data_cosmos`. It's a relatively low-impact crate, but I'm open to removing it and going back to a series of `#[cfg(...)]`-attributed definitions if that's preferred. (cc @heaths )